op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```
@@ -0,0 +1,251 @@
+
+- [Runbook: Incident Response for Faultproof Withdrawals](#runbook-incident-response-for-faultproof-withdrawals)
+  - [Overview](#overview)
+  - [[!IMPORTANT] Disclaimer: work in progress!](#-disclaimer-work-in-progress)
+  - [Automated runbooks](#automated-runbooks)
+  - [Alerts](#alerts)
+    - [faultproof-withdrawal-forgery-detected](#faultproof-withdrawal-forgery-detected)
+      - [Alert Description](#alert-description)
+      - [Triage Phase](#triage-phase)
+    - [faultproof-potential-withdrawal-forgery-detected](#faultproof-potential-withdrawal-forgery-detected)
+        - [Alert Description](#alert-description-1)
+      - [Triage Phase](#triage-phase-1)
```

Markdownlint

Expected: 8; Actual: 6
Inconsistent indentation for list items at the same level

(MD005, list-indent)